### PR TITLE
ViewModel should initialize with some data pre-set. #5

### PR DESCRIPTION
**Pull request by KassandraA**
Done it.

### CODE REVIEW

File: source/view-model.js [hunk 1]
```diff
@@ -4,14 +4,23 @@
...
 function ViewModel(){
     this.game = null;
 
     this.getPlayersArray = function () {
         let array = [];
         let factory = new Factory();
-        array.push(factory.createPlayer('human', 'First', 'X'));
-        array.push(factory.createPlayer('computer', 'Computer', 'O'));
+        let gamePreset = new GamePreset();
+        for(let i = 0; i < gamePreset.players.length; i++){
```
Comment (aannenko) [inline source/view-model.js]: Can we replace `for` with `gamePreset.players.forEach` and then simplify the internals of the cycle?

File: source/view-model.js [hunk 2]
```diff
@@ -4,14 +4,23 @@
 
 /*********************** View-Model **********************/
 "use strict";
+function GamePreset(){
```
Comment (aannenko) [inline source/view-model.js:7]: Can we put informational (non-functional) objects in the same chunk of code? GamePreset and GameInfo have the same purpose and can be placed in the code one after the other.

**Review by aannenko — CHANGES_REQUESTED** [review]

File: source/view-model.js (hunk 2 above)
Comment (KassandraA) [reply source/view-model.js:7]: Done

File: source/view-model.js (hunk 1 above)
Comment (KassandraA) [reply source/view-model.js]: Done

**Pull request closed without merging**